src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,194 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 1 << 4;
+    private static final int MAXIMUM_CAPACITY = 1 << 30;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int tableSize = 0;
+    private int threshold;
+
+    static class Node<K, V> {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line before the Node class declaration.